### PR TITLE
Fm view picture

### PR DESCRIPTION
**Pull request by pranotoism**
### Detailed description of changes
NEW FMViewPicture
FMViewPicture test
FMViewPicture storybook
...

### Recommendations on how to test the changes

...

### Checklist for developer

- [x] The code builds clean without any errors or warnings
- [x] The code does not contain commented out code
- [x] The code does not log anything to console
- [x] I have added unit test(s) to cover new code and successfully executed ran it
- [x] I have thoroughly tested the new code and any adjacent features it may affect

### Link to related issue(s)

...

#### Before

...

#### After
Screen Shot 2022-03-14 at 9 05 51 AM
Screen Shot 2022-03-14 at 9 05 59 AM

...

### CODE REVIEW

File: stories/Components/Molecules/FMViewPicture/FMViewPicture.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+const story = {
+  title: "Molecules/View Picture",
+  component: FMViewPicture,
+  argTypes: {
+    children: { control: "text" },
+  },
+};
+
+export default story;
```
Comment (bryantobing12) [inline stories/Components/Molecules/FMViewPicture/FMViewPicture.stories.tsx:15]: We are now use this convention. kindly update. Thanks
```suggestion
const meta: Meta = {
  title: "Molecules/View Picture",
  component: FMViewPicture,
  argTypes: {
    children: { control: "text" },
  },
};

export default meta;
```

File: src/Components/Molecules/FMViewPicture/index.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      >
+        {label}
+      </FAButton>
+      <FMBackdropImage
+        testID="backdrop-image-picture"
+        sx={{ zIndex: theme => theme.zIndex.drawer + 1 }}
+        open={open}
+        handleClose={handleClose}
+      >
+        <img src={img.src} alt={img.alt} />
```
Comment (bryantobing12) [inline src/Components/Molecules/FMViewPicture/index.tsx:40]: we may want to use next/image for optimization. that won't be possible with this approach. we can ignore if that's not the case

File: src/Components/Molecules/FMViewPicture/index.tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
+import React, { memo, useState } from "react";
+import { Box } from "@mui/material";
+
+import { IFMViewPicture } from "./FMViewPicture.interface";
+import { FAAvatar, FAButton } from "@Atoms";
+import { FMBackdropImage } from "@Molecules/FMBackdropImage";
```
Comment (bryantobing12) [inline src/Components/Molecules/FMViewPicture/index.tsx:6]: Kindly update the import order. 

```suggestion
import React, { memo, useState } from "react";
import Box from "@mui/material/Box";

import { FAAvatar, FAButton } from "@Atoms";
import { FMBackdropImage } from "@Molecules";

import { IFMViewPicture } from "./FMViewPicture.interface";
```

**Review by muhammad-farhan64 — APPROVED** [review]

File: stories/Components/Molecules/FMViewPicture/FMViewPicture.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+import { FMViewPicture } from "@Molecules";
+import { Meta } from "@storybook/react";
```
Comment (bryantobing12) [inline stories/Components/Molecules/FMViewPicture/FMViewPicture.stories.tsx:4]: Fix import order

File: src/Components/Molecules/FMViewPicture/FMViewPicture.interface.ts [hunk 5]
```diff
@@ -0,0 +1,14 @@
+import { ReactChild } from "react";
+import { IFAAvatarProps } from "@Atoms/FAAvatar/FAAvatar.interface";
+import { IFAButtonProps } from "@Atoms/FAButton/FAButton.interface";
```
Comment (bryantobing12) [inline src/Components/Molecules/FMViewPicture/FMViewPicture.interface.ts:3]: don't use absolute path import. kindly update to relative import. https://hzngroup.slack.com/archives/C0318NE813M/p1645763123950249

**Review by bryantobing12 — CHANGES_REQUESTED** [review]
kindly update before merge. Thanks

**Review by bryantobing12 — APPROVED** [review]

**Pull request merged**